dipu/torch_dipu/csrc_dipu/aten/ops/CustomFallbackFunctionsForAmpGradScaler.cpp

```diff
@@ -52,6 +52,15 @@ void custom_fallback_dipu__amp_foreach_non_finite_check_and_unscale_(
   }
 }
 
+// growth_tracker in torch 2.1 is a scalar tensor. in 2.0 is a size=1 tensor.
+void inline set_growth_tracker(at::Tensor& growth_tracker, int value) {
```

Comment:
move to anonymous namespace
```suggestion
void set_growth_tracker(at::Tensor& growth_tracker, int value) {
```